src/everest/config/control_config.py

```diff
@@ -180,16 +172,14 @@ def ropt_perturbation_type(self) -> PerturbationType:
     def ropt_control_type(self) -> VariableType:
         return VariableType[self.control_type.upper()]
 
-    @model_validator(mode="after")
-    def check_for_autoscale_flag(self) -> Self:
-        if self.auto_scale is not None:
-            logger.warning(
-                "auto_scale is deprecated for everest "
-                "controls, and is on by default. Please remove it"
-                "from the Everest config."
+    @model_validator(mode="before")
+    @classmethod
+    def check_for_autoscale_flag(cls, values: dict[str, Any]) -> dict[str, Any]:
+        if "auto_scale" in values:
+            raise ValueError(
+                "auto_scale is deprecated for everest controls, and is on by default."
```

Comment:
It seems harsh to break old configs, but assume this needs to be like this to make sure people using the deprecated key are made aware that the behaviour has changed.

Reply:
Agree that it is a bit harsh, so there is a bit of a balance here. We should at least remove it from the schema so it does not confuse users. Could replace the ValueError with a ConfigWarning, but a logging statement is not visible enough.